packages/@okta/vuepress-site/docs/guides/migrate-to-oie/index.md

```diff
@@ -0,0 +1,155 @@
+
+For most users of the [custom domain](/docs/guides/custom-url-domain/overview/) feature, there are no other changes needed. The default template will detect and use the Okta Identity Engine automatically.
+
+However if you have [modified the template](https://developer.okta.com/docs/guides/style-the-widget/style-okta-hosted/) in certain ways, such as to perform redirects or set cookies, these modificiations may not be compatible with the Okta Identity Engine. In particular, use of these methods and objects may not work with OIE:
+
+- setCookieAndRedirect
+- sessionToken
+
+In the default template, the `success` callback from `renderEl` is being handled by `OktaUtil.completeLogin`. This method correctly handles the authentication flow and we recommend using it.
+
+For reference, here is the default template:
+
+```html
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
+<html>
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="robots" content="none" />
+
+    <title>{{pageTitle}}</title>
+    {{{SignInWidgetResources}}}
+</head>
+<body>
+    <div class="login-bg-image" style="background-image: {{bgImageUrl}}"></div>
+    <div id="okta-login-container"></div>
+
+    <!--
+        "OktaUtil" defines a global OktaUtil object
+        that contains methods used to complete the Okta login flow.
+     -->
+    {{{OktaUtil}}}
+
+    <script type="text/javascript">
+        // "config" object contains default widget configuration
+        // with any custom overrides defined in your admin settings.
+        var config = OktaUtil.getSignInWidgetConfig();
+
+        // Render the Okta Sign-In Widget
+        var oktaSignIn = new OktaSignIn(config);
+        oktaSignIn.renderEl({ el: '#okta-login-container' },
+            OktaUtil.completeLogin,
+            function(error) {
+                // Logs errors that occur when configuring the widget.
+                // Remove or replace this with your own custom error handler.
+                console.log(error.message, error);
+            }
+        );
+    </script>
+</body>
+</html>
+```
+
+### Embedded signin widget
+
+Set the option `useInteractionCodeFlow` to `true` on the object passed to the Sign-In Widget constructor. This will enable Okta Identity Engine for the widget. Both the authorization server and the application must have the [interaction code](#enable-interaction-code-grant) grant type enabled.
+
+> **Note:** Your code may break if it is calling the `renderEl` method and expects `sessionToken` or `session.setCookieAndRedirect` on the response object. Instead of `renderEl`, we recommend calling the `showSignInToGetTokens` method. This method receives and returns tokens without any browser redirect.
+
+```javascript
+var signIn = new OktaSignIn(
+  {
+    baseUrl: 'https://{yourOktaDomain}',
```

Comment:
@shuowu - how are you handling passing `issuer` to the idx code?  I know we can use the baseUrl to use their org AS, but IIRC there are some claims they won't be able to get unless they use their custom AS (for those that have them).

Depending on Shuo's answer, we may want to mention the use of issuer here.

Reply:
I think people usually pass the issuer as:
```
{
  baseUrl: '...',
  authParams: {
    issuer: '...
  }
}
```

Reply:
`baseUrl/oauth2/default` is set as the issuer if it is not provided. To use org as issuer you have to explicitly pass it in under authParams